css/lookbook.scss

```diff
@@ -0,0 +1,93 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+@import url(https://fonts.googleapis.com/css?family=Libre+Baskerville);
+@import 'common';
+@import 'styleguide';
```

Comment:
you just need to import common, common imports styleguide. also delete the @import font lines, common takes care of it for you